.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -1,19 +1,21 @@
 ---
 name: Feature Request
 about: Suggest an idea for for a new feature for Betaflight
-
+labels: "Feature Request"
```

Comment:
I think these auto-classification labels should be different from the manually applied, existing labels `BUG` and `Feature Request`. Proposal: `Template: Bug` / `Template: Feature Request` - this will still allow us to manually apply the other labels for issues that qualify.